### DISCUSSION TITLE
Making Custom Mob 1.21.4

### DISCUSSION THREAD

**Discussion by JCS-Mecabricks** [FabricMC/fabric-api#4277] [Mod Dev Support]
### Description

What is the command for a custom mob model in 1.21.4. In 1.21 in was public class CustomModel extends SinglePartEntityModel<CustomEntity> but now I don't know what it is.

**Comment by Jauchlander** [comment]
Could you let me know if you found the replacement?

**Reply by JCS-Mecabricks** [reply]
I haven't.

**Reply by Jauchlander** [reply]
I have been looking into it... There are MANY changes... Not only that. The EntityModel class itself changed. As lots of other classes... 
I think that's why most of the mods made for 1.21.1 are not yet updated to 1.21.X (.2, .3, .4 and now .5)

**Reply by Jauchlander** [reply]
Here: https://fabricmc.net/2024/10/14/1212.html
